init_users_mysql.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Database connection details
+DB_HOST="localhost"
+DB_PORT="3307"
+DB_NAME="puresynth"
+DB_USER="pUser"
+DB_PASS="pSynth"
+
+# Function to insert a user
+insert_user() {
+    local username="$1"
+    local password="$2"
+    local email="$3"
+    local role="$4"
+
+    local sql="
+        INSERT INTO roles(name) VALUES ('$role') ON DUPLICATE KEY UPDATE name=name;
+        INSERT INTO user(username, password, email) VALUES ('$username', '$password', '$email');
+        SET @user_id = LAST_INSERT_ID();
+        INSERT INTO user_roles(user_id, role_id) SELECT @user_id, id FROM roles WHERE name='$role';
+        "
+
+    echo "$sql" | mysql -h $DB_HOST -P $DB_PORT -u $DB_USER -p$DB_PASS $DB_NAME
+}
+
+# Insert users
```

Comment:
This is another doubt that I was having.
the passwords, in reality, are super simple. However, I passed through the BCryptPasswordEncoder because all the other persisted users have the password stored as an encrypted password. 
I will modify it